test/ConfigProviderTest.php

```diff
@@ -10,23 +10,50 @@
 use PHPUnit\Framework\TestCase;
 use Zend\Di\ConfigProvider;
 use PHPUnit\Framework\Constraint\IsType;
+use Zend\Di\CodeGenerator\InjectorGenerator;
+use Zend\Di\InjectorInterface;
+use Zend\Di\ConfigInterface;
```

Comment:
Please sort imports in alphabetical order (it will be also done automatically in future by zend-coding-standard)